src/compiler/program.ts

```diff
@@ -3019,6 +3019,13 @@ export function createProgram(rootNamesOrOptions: readonly string[] | CreateProg
                     case SyntaxKind.TypeAliasDeclaration:
                         diagnostics.push(createDiagnosticForNode(node, Diagnostics.Type_aliases_can_only_be_used_in_TypeScript_files));
                         return "skip";
+                    case SyntaxKind.MethodDeclaration:
+                    case SyntaxKind.FunctionDeclaration:

```

Comment:
Accessor and constructor ?

Reply:
Accessors can't be overloaded AFAICT because that doesn't make sense. I missed constructors; I thought they are not overloadable.